1-js-and-async-programming/1-javascript-fundamentals/1-time-difference.js

```diff
@@ -16,6 +16,15 @@ timeDifference(7200, 3400); // Expected output: "01:03:20"
 
 */
 
-const timeDifference = (a, b) => {};
+const timeDifference = (a, b) => {
+    if( a < 0 || b < 0) throw new Error('');
+    if( a === 0 && b === 0) return '00:00:00';
+    const differenceInSeconds = Math.abs(a - b);
+    const hours = Math.floor(differenceInSeconds/3600);
+    const minutes = Math.floor(differenceInSeconds%3600/60);
+    const seconds = differenceInSeconds%3600%60;
+    const padWidthZero = (num) => (num < 10) ? `0${num}` : String(num);
```

Comment:
We can leverage the padStart built-in function of a String type
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart
```suggestion
    const padWidthZero = String(num).padStart(2, '0')
```